README.md

```diff
@@ -107,11 +109,19 @@ are supported using the folder structure above.
   <summary> <b> HDD </b> </summary>
 <p>
 
-For PS2, 48-bit LBA internal HDDs up to 2TB are supported. HDD should be
+For 48-bit LBA internal HDDs 2TB and under, the HDD should be
 formatted with the APA partition scheme. OPL will create the `+OPL` partition on the HDD.
 To avoid this, you can create a text file at the location `hdd0:__common:pfs:OPL/conf_hdd.txt`
 that contains the preferred partition name (for example `__common`).
 
+You are able to format drives with the exFAT filesystem if you would like to avoid the 2TB limitation.
+In order for the exFAT configuration to operate successfully, there must not be any fragmentation with the files.
+Files should be added contiguously or synchronously to avoid fragmentation.  
+On Windows, this is the default behavior when copying files to another location.  On Linux, you may have to specify a flag when you mount the drive to achieve synchronous behavior.
+Please make note of this caveat when you are troubleshooting any issues.
```

Comment:
Not true. 64 fragments are supported, same as all other BDM devices. See readme part about USB.